bliss/simulator/simulated_dataset.py

```diff
@@ -44,22 +46,26 @@ def __init__(
         self.background.requires_grad_(False)
 
         assert survey.psf is not None, "Survey psf cannot be None."
-        assert survey.bands is not None, "Survey bands cannot be None."
+        assert survey.pixel_shift is not None, "Survey pixel_shift cannot be None."
+        assert (
+            survey.physical_to_nelec_dict is not None
+        ), "Survey physical_to_nelec_dict cannot be None."
         self.image_decoder = ImageDecoder(
             psf=survey.psf,
             bands=survey.BANDS,
             pixel_shift=survey.pixel_shift,
-            nmgy_to_nelec_dict=survey.nmgy_to_nelec_dict,
+            physical_to_nelec_dict=survey.physical_to_nelec_dict,
```

Comment:
I'd call this `flux_calibration` too. (I don't think people will understand that "physical" refers to "physical units of flux")

Reply:
Done.